babble/nlp/engine.py

```diff
@@ -131,28 +131,54 @@ def _load_entities(self) -> Dict[str, Dict]:
                 entities[element.get("name")] = element
         return entities
 
+    def _get_best_match(self, alternatives: List[Understanding]):
+        alternatives_len = {
+            alternative: alternative.required_matched_classifiers
+            for alternative in alternatives
+        }
+        max_len = max(alternatives_len.values())
+        longest_alternatives = [
+            key for key in alternatives_len.keys() if alternatives_len[key] == max_len
+        ]
+        if len(longest_alternatives) == 1:
+            return longest_alternatives[0]
+        else:
+            alternatives_validity = {
+                alternative: alternative.validity()
+                for alternative in longest_alternatives
+            }
+            log.info(
```

Comment:
Maybe better a debug output. Once we know that this works we don't need this info in our logs.